build/Targets/GenerateCompilerInternals.targets

```diff
@@ -45,125 +46,99 @@
+      <GeneratedSyntaxModelGetText>@(SyntaxGetTextDefinition -&gt; '$(IntermediateOutputPath)\%(FileName)%(Extension).Generated$(DefaultLanguageSourceExtension)')</GeneratedSyntaxModelGetText>
     </PropertyGroup>
 
     <Delete Files="$(GeneratedSyntaxModelGetText)" />
   </Target>
 
-  <Target
-    Name="GenerateSyntaxModelTests"
-    Inputs="@(SyntaxTestDefinition);$(VBSyntaxGeneratorToolPath);$(CSharpSyntaxGeneratorToolPath)"
-    Outputs="@(SyntaxTestDefinition -> '$(IntermediateOutputPath)\%(FileName)%(Extension).Generated$(DefaultLanguageSourceExtension)')"
-    Condition="'@(SyntaxTestDefinition)' != '' AND '$(BuildingProject)' == 'true' AND ('$(Language)' == 'VB' or '$(Language)' == 'C#')"
-  >
+  <Target Name="GenerateSyntaxModelTests"
+          Inputs="@(SyntaxTestDefinition);$(VBSyntaxGeneratorToolPath);$(CSharpSyntaxGeneratorToolPath)"
+          Outputs="@(SyntaxTestDefinition -&gt; '$(IntermediateOutputPath)\%(FileName)%(Extension).Generated$(DefaultLanguageSourceExtension)')"
+          Condition="'@(SyntaxTestDefinition)' != '' AND '$(BuildingProject)' == 'true' AND ('$(Language)' == 'VB' OR '$(Language)' == 'C#')">
     <PropertyGroup>
       <SyntaxGenerator Condition="'$(Language)' == 'VB'">"$(CoreRunExe)" "$(VBSyntaxGeneratorToolPath)"</SyntaxGenerator>
       <SyntaxGenerator Condition="'$(Language)' == 'C#'">"$(CoreRunExe)" "$(CSharpSyntaxGeneratorToolPath)"</SyntaxGenerator>
-      <GeneratedSyntaxModelTests>@(SyntaxTestDefinition -> '$(IntermediateOutputPath)\%(FileName)%(Extension).Generated$(DefaultLanguageSourceExtension)')</GeneratedSyntaxModelTests>
+      <GeneratedSyntaxModelTests>@(SyntaxTestDefinition -&gt; '$(IntermediateOutputPath)\%(FileName)%(Extension).Generated$(DefaultLanguageSourceExtension)')</GeneratedSyntaxModelTests>
     </PropertyGroup>
 
-    <Exec
-      Command='$(SyntaxGenerator) &quot;@(SyntaxTestDefinition)&quot; &quot;$(GeneratedSyntaxModelTests)&quot; /test'
-      Outputs="$(GeneratedSyntaxModelTests)"
-    >
+    <Exec Command="$(SyntaxGenerator) &quot;@(SyntaxTestDefinition)&quot; &quot;$(GeneratedSyntaxModelTests)&quot; /test"
+          Outputs="$(GeneratedSyntaxModelTests)">
       <Output TaskParameter="Outputs" ItemName="FileWrites" />
     </Exec>
 
   </Target>
 
-  <Target
-    Name="CleanSyntaxModelTests"
-    Condition="'$(Language)' == 'VB' or '$(Language)' == 'C#'"
-  >
+  <Target Name="CleanSyntaxModelTests" Condition="'$(Language)' == 'VB' OR '$(Language)' == 'C#'">
     <PropertyGroup>
-      <GeneratedSyntaxModelTests>@(SyntaxTestDefinition -> '$(IntermediateOutputPath)\%(FileName)%(Extension).Generated$(DefaultLanguageSourceExtension)')</GeneratedSyntaxModelTests>
+      <GeneratedSyntaxModelTests>@(SyntaxTestDefinition -&gt; '$(IntermediateOutputPath)\%(FileName)%(Extension).Generated$(DefaultLanguageSourceExtension)')</GeneratedSyntaxModelTests>
```

Comment:
I don't agree with this change.  The previous use of `->` is how MSBuild recommends this be written.  Even if it's illegal XML it's the idiomatic style. 

https://msdn.microsoft.com/en-us/library/ms171476.aspx